app/controllers/admin/project_logs_controller.rb

```diff
@@ -11,13 +11,14 @@ def show
   def new
     @log = ProjectLog.new(project_step_id: params[:step_id])
     authorize_with_parents
-    render layout: false
+    # render layout: false
+    render partial: "admin/logs/new"
   end
 
   def edit
     @log = ProjectLog.find(params[:id])
-    authorize_with_parents
```

Comment:
@hooverlunch: The edit action loads the entire loan inside the modal. I think I've encountered this before with another part of our application (loading the parent instead of the specific object). Let me know if you know why this happening. I removed authorize_with_parents only to see if that would fix the problem, so let's keep it in, if it is needed. Thanks.